tools/scripts/stm32.mk

```diff
@@ -19,7 +19,7 @@ $(error $(ENDL)$(ENDL)STM32CUBEMX not defined or not found at default path /opt/
 endif # STM32CUBEMX check
 
 # Locate the compiler path under STM32CubeIDE plugins directory
-COMPILER_BIN = $(realpath $(dir $(call rwildcard, $(STM32CUBEIDE)/plugins, *arm-none-eabi-gcc)))
+COMPILER_BIN = $(realpath $(dir $(call rwildcard, $(STM32CUBEIDE)/plugins, *arm-none-eabi-gcc)))/arm-none-eabi-gcc
```

Comment:
something is not right about adding /arm-none-eabi-gcc to the path

Reply:
I'll separate it out into a different variable so and check to see if that still works

Reply:
So this change should be removed from this commit (you are doing the necessary things in the 3rd commit)

Reply:
This has not been fixed (simply remove this change from this commit)